store.html

```diff
@@ -24,6 +24,8 @@
 
     <section class="store_main_section">
         <div class="form_container">
+            <fieldset class="form_container">
```

Comment:
fieldset is best used around a group of radio buttons for accessability see https://syllabus.codeyourfuture.io/html-css/week-2/lesson#radio-input-fields https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset